.github/workflows/actions/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+yum install -y udunits2-devel
+PYTHONS=("cp36-cp36m" "cp37-cp37m" "cp38-cp38" "cp39-cp39")
+
+cp /usr/include/udunits2/* /usr/include/
+
+SITECFG=cf_units/etc/site.cfg
+echo "[System]" > $SITECFG
+echo "udunits2_xml_path = /usr/share/udunits/udunits2.xml" >> $SITECFG
```

Comment:
@ocefpaf If you now rebase your PR with `main`, given that #191 is merged, I think you need to replace `lines 6-10` with the following:
```bash
export UDUNITS2_INCDIR="/usr/include/udunits2"
export UDUNITS2_LIBDIR="/usr/lib64"
export UDUNITS2_XML_PATH="/usr/share/udunits/udunits2.xml"
```